src/opera/util/input_validation.py

```diff
@@ -131,7 +131,7 @@ def validate_slc_s1_inputs(runconfig, logger, name):
             logger.critical(name, ErrorCode.INVALID_INPUT, error_msg)
 
 
-def get_burst_id_set(input_file_group, logger, name):
+def check_disp_s1_ancillary_burst_ids(input_file_group, logger, name):
```

Comment:
My intent was not to have this function renamed, but the one below it (check_amp
_mean_geometry_burst_ids). As-is, this function now has a misleading name (is not specific to ancillaries), and the one below is still too specific (could be applied to any hypothetical ancillary group, not just the amp/mean/geometry files).

Reply:
Makes sense haha.